src/ctype/CType.spec.ts

```diff
@@ -73,6 +58,6 @@ describe('CType', () => {
   })
   it('verifies the claim structure', () => {
     expect(claimCtype.verifyClaimStructure(claim)).toBeTruthy()
-    expect(claimCtype.verifyClaimStructure(!claim)).toBeFalsy()
```

Comment:
I noticed this wasn't doing what I wanted, because of the checks in place regarding the schema interfaces it was hard to find a way around without calling errors in the above code. So, I found a hacky way of fixing it. 

Please, let me know if this is ok.

Reply:
I think it would be best to modify the claim, so that it actually does not match the ctype